PENDING_PENDING_RELEASE_NOTES.md

```diff
@@ -0,0 +1,11 @@
+
+## Version 15.13.0
+_Date TBD_
+
+### Minor Features
+* Add new chart: New vs. Incumbent Developer chart
```

Comment:
missing space, also since we did the release this should just be one PENDING right?